### PR TITLE
Update to Venafi support page.

### PR DESCRIPTION
**Pull request by davidnoyes**

### CODE REVIEW

**Comment by davidnoyes** [comment]
Please hold on merges pending vendor review
Thanks

**Comment by davidnoyes** [comment]
Ok. The vendor approves. Happy to merge after review.

File: content/pages/support.mdx [hunk 1]
```diff
@@ -5,13 +5,4 @@ export const meta = {
     heading: 'Commercial Support',
     description: 'Need a little more help?'
   },
-  intro: {
-    heading: 'Venafi',
-    description:
-      'Venafi is the principal maintainer of the cert-manager project and works directly with the CNCF. Commercial support for cert-manager is available as part of Venafi\'s Long Term Support (LTS) offering which includes access to expertise to help organisations scale effectively and securely with cert-manager.',
-    cta: {
-      logo: '/images/venafi-lts-support.png',
```
Comment (SgtCoDFish) [inline content/pages/support.mdx:13]: suggestion: It looks like this image isn't used anywhere else and we're removing it here - could we remove it? It's at `public/images/venafi-lts-support.png`

Comment (davidnoyes) [reply content/pages/support.mdx:13]: ### You are correct. Removed in the latest push

**Review by SgtCoDFish — APPROVED** [review]
/lgtm
/approve

Thanks 🚀

**Pull request merged**